at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb

```diff
@@ -0,0 +1,42 @@
+class StringFormat
+  class << self
+  def human_name(string)
+    if string.is_a? String
+      i = 0
+      out = ''
+      string.strip!
+      string.gsub!(/[^A-Za-z ]/, '')
+      string.each_char do |x|
+        if string[i - 1] == ' ' && x != ' ' || i.zero?
+          _a = x.ord - 32
+          out += _a.chr
+        else
+          out += x
+        end
+        i += 1
+      end
+      out
+    else
+      nil
+    end
+  end
+
+  def uniq(string)
+    hash = {}
+    string.each_char { |_str| hash[_str] = ((hash[_str] || 0) + 1) }
+    hash.keys.inject { |out, i| out + i }
+  end
+
+  def only_letter?(string)
```

Comment:
Check only letters not only one character.

Reply:
only_letter?(string) Trả về `true` nếu chỉ toàn chữ hoa hoặc thường. `false` cho tất cả trường hợp còn lại

Reply:
@at-luantran

Reply:
dạ e mới fix lại ạ. Anh riview lại giúp em với ạ.